src/pyxplor/pyxplor.py

```diff
@@ -0,0 +1,62 @@
+def plot_binary(input_df: pd.DataFrame,
```

Comment:
See *consistency* comments on @arturoboquin 's PR (https://github.com/UBC-MDS/PyXplor/pull/15/files)...

I like the `plot_...` naming so @iris0614 @arturoboquin we could consider all adopting that (and could decide as well on common parameter naming / structure)